pkgs/os-specific/darwin/apparency/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchurl
+, stdenv
+, undmg
+}:
+
+stdenv.mkDerivation {
+  pname = "apparency";
+  version = "1.5.1";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20230815073821/https://www.mothersruin.com/software/downloads/Apparency.dmg";
+    sha256 = "sha256-JpaBdlt8kTNFzK/yZVZ+ZFJ3DnPQbogJC7QBmtSVkoQ=";
+  };
+
+  nativeBuildInputs = [ undmg ];
+
+  sourceRoot = "Apparency.app";
+
+  installPhase = ''
+    mkdir -p "$out/Applications/Apparency.app"
+    cp -R . "$out/Applications/Apparency.app"
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.mothersruin.com/software/Apparency/";
+    description = "The App That Opens Apps";
+    platforms = platforms.darwin;
+    license = licenses.unfreeRedistributable;
+    maintainers = with maintainers; [ Enzime ];
```

Comment:
```suggestion
    license = lib.licenses.unfreeRedistributable;
    maintainers = with lib.maintainers; [ Enzime ];
    platforms = lib.platforms.darwin;
    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
```

Reply:
Noob question from a non-Darwin user: this package runs in both x86 and arm darwin?

Reply:
Yeah it does 👍